app/assets/stylesheets/govuk_publishing_components/components/_document-list.scss

```diff
@@ -32,6 +32,10 @@
   float: left;
   list-style: none;
   padding-right: $gutter-two-thirds;
+
+  .direction-rtl & {
+    float: right;
+  }
```

Comment:
Does this also need to have the padding-right changed to padding-left? The indenting of the date doesn't look quite right...

![screen shot 2018-07-17 at 12 29 19](https://user-images.githubusercontent.com/861310/42814748-1ee86c7e-89bd-11e8-955f-57fdaa18d6ff.png)